app/cli/internal/action/workflow_update.go

```diff
@@ -30,18 +30,19 @@ func NewWorkflowUpdate(cfg *ActionsOpts) *WorkflowUpdate {
 }
 
 type NewWorkflowUpdateOpts struct {
```

Comment:
this is not from your PR, but at first it seems that I had a brain fart when naming this struct probably because I copied it from the creation flow. `NewWorkflowUpdateOpts` should probably be called `WorkflowUpdateOpts`

Reply:
Let me do the change!